### ISSUE TITLE
Communotron 16 has extremely low non-RT value

### ISSUE THREAD

**Issue by Bornholio** [KSP-RO/RealismOverhaul#1686]
Communotron 16 has range value of 32 on comnet.

\GameData\RealismOverhaul\RO_SuggestedMods\Squad\RO_Squad_Communication.cfg
line: 96        @antennaPower = 32

Suggest 8Mm  giving 632Mm vs L2DSN
line: 96        @antennaPower = 8000000

**Comment by PhineasFreak** [comment]
Did you use CBK while testing?

https://github.com/KSP-RO/RealismOverhaul/pull/1634

The ranges of the antennae under CommNet are **exactly** the same as the ranges under RT **for the respective tracking station range**.

**Comment by Bornholio** [comment]
CBK installed while testing, the condition I noticed in was while at moon, released impactor that had two sputnik antenna, main craft had 4xCommu16's. Main craft then lost communications as the impactor left range.

**Comment by PhineasFreak** [comment]
Something that i just remembered: did you enforce a reload of the MM cache? If not, can you do so and report back with the same craft file?

**Comment by Bornholio** [comment]
> Edit: foget it, it is probably the antenna exponent(s) that have been set incorrectly...

thoughts on correcting it? the only comnet exponent I saw was the multiple antenna multiplier
```
    @MODULE[ModuleDataTransmitter]
    {
        @antennaType = DIRECT
        @antennaCombinable = True
        %antennaCombinableExponent = 1
        @antennaPower = 32
        @packetInterval = 1.0
        @packetSize = 0.768
        @packetResourceCost = 0.0005
    }
```

versus for the Surface Antenna

```
    @MODULE[ModuleDataTransmitter]
    {
        @antennaType = DIRECT
        @antennaCombinable = True
        %antennaCombinableExponent = 1
        @antennaPower = 8000000
        @packetInterval = 1.0
        @packetSize = 1.28
        @packetResourceCost = 0.004
    }
```

**Comment by PhineasFreak** [comment]
Set the antenna power from 32 to 3200 (or even 320000 if you want ease of access even with the LV 1 tracking station).

**Comment by Bornholio** [comment]
[img]http://i.imgur.com/QsBqM9j.png[/img]
[img]http://i.imgur.com/JBFPonP.png[/img]
neither setting (3,200 or 320,000) are sufficient to reach Earth from the Moon

**Comment by PhineasFreak** [comment]
The ranges indicated in the part module are wrong. If the antenna range was 320000 and the Tracking Station level is 2 (500 Gm) then:

```
R = SQRT(AntennaRange * StationRange) = SQRT(320000 * 500000000000) = 400000000 m (400 Mm)
```

With a range value of 3200 you will get 40 Mm (multiply by 100 to get a ten fold increase in range).

Reload the MM cache and try again.

**Comment by Bornholio** [comment]
OK let me make a table, it comes from the current /master/GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Communication.cfg
Commu16-S  @antennaPower = 8000000               
Comm16        @antennaPower = 32
DT-M1           @antennaPower = 57800000000
HG55             @antennaPower = 231200000000
HG5               @antennaPower = 288800000000
RA2                @antennaPower = 672800000000
Com8888      @antennaPower = 125000000000
RA15             @antennaPower = 8000000000000
RA100           @antennaPower = 32000000000000    

If the value for Commu16 is not clearly wrong I will close this issue. (these values are in CommNet ModuleDataTransmitter sections).

**Comment by PhineasFreak** [comment]
Calculating and plotting the current ranges against the relative tracking station ranges you get:

| Antenna Name | Station Level | Antenna Range |
| :----------------: | :-------------: | :------------------: |
| Communotron 16-S | Level 1 | 200 Mm |
| Communotron 16 | Level 2 | 4 Mm |
| DTS-M1 | Level 2 | 170 Gm |
| Communotron HG-55 | Level 2 | 340 Gm |
| HG-5 | Level 2 | 380 Gm |
| RA-2 | Level 2 | 580 Gm |
| Communotron 88-88 | Level 3 | 2.5 Tm |
| RA-15 | Level 3 | 20 Tm |
| RA-100 | Level 3 | 40 Tm |

The problem with the Communotron 16 is that i was expecting the range to be multiplied, just like with RT, but it did not work. So, the range was low by a factor of 100.

**Comment by Bornholio** [comment]
CommNet Communotron 16 range fix #1688
